contracts/Allocator.sol

```diff
@@ -61,24 +61,31 @@ contract Allocator is Killable, Ownable, UsingConfig, Withdrawable {
 			pendingIncrements[_metrics] == true,
 			"Not asking for an indicator"
 		);
-		address property = metrics.property();
-		uint256 share = market.issuedMetrics() /
-			MetricsGroup(config().metricsGroup()).totalIssuedMetrics();
+		Policy policy = Policy(config().policy());
+		Lockup lockup = Lockup(config().lockup());
+		uint256 totalIssuedMetrics = MetricsGroup(config().metricsGroup())
+			.totalIssuedMetrics();
+		uint256 share = market.issuedMetrics() / totalIssuedMetrics;
 		uint256 period = block.number -
 			lastAllocationBlockEachMetrics[_metrics];
-		uint256 allocationPerBlock = _value / period;
+		uint256 lockupValue = lockup.getTokenValue(
+			msg.sender,
+			metrics.property()
+		);
```

Comment:
In here, lockupValue should indicate the total lockup amount of this property.

Reply:
This expression indicates how much `msg.sender` (a Market) is locking up to this Property.